Sources/Slacket/Slacket.swift

```diff
@@ -13,15 +13,21 @@ import LoggerAPI
 
 import libc
 
-struct ExternalServerConfig: URLType {
-
-    let host: String = "slacket.link"
-}
-
-struct InternalServerConfig: URLType {
-
-    let host: String = "localhost"
-    let port: Int? = 8090
+struct ServerConfig: URLType {
+    
```

Comment:
ServerConfig should be a singleton, to emphasise there is only one version of config